flow-server/src/main/java/com/vaadin/flow/router/InternalServerError.java

```diff
@@ -137,10 +137,9 @@ private void checkLogBinding() {
                             + "As a result the logger doesn't do any real logging. "
                             + "Add some binding as a dependency to your project. "
                             + "See details ");
-            logInfo.getStyle().set("marginTop", "10px");
-            logInfo.getStyle().set("marginBottom", "10px");
+            logInfo.getStyle().setMargin("10px 0");
```

Comment:
this could be, right?:
```suggestion
            logInfo.getStyle().setMargin("10px 10px");
```

Reply:
Why would we want to change the implementation? Then there would be margin also on left and right side of the logInfo element?

Reply:
I missed/missee this implementation, this is good like this.
First (before coffee) I thought we changed it to `marginTop: 10px, marginBottom: 0` with your code change, but then looked better, and indeed, this is the correct form this will be the same as before.

Reply:
Where can I send more coffee to you 😁

Reply:
Maybe it is a good value proposition, just give miss review comments and then a buy me a coffee link after they are annoyed and want you to stop :))). Smart! ;)